ui/tests/integration/components/ttl-picker2-test.js

```diff
@@ -0,0 +1,117 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render, click, fillIn } from '@ember/test-helpers';
+import sinon from 'sinon';
+import hbs from 'htmlbars-inline-precompile';
+
+module('Integration | Component | ttl-picker2', function(hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders time and unit inputs when TTL enabled', async function(assert) {
+    let changeSpy = sinon.spy();
+    this.set('onChange', changeSpy);
+    await render(hbs`
+      <TtlPicker2
+        @onChange={{onChange}}
+        @enableTTL={{true}}
+      />
+    `);
+
```

Comment:
Not a big deal, but a line space here, and not a space in the test below it.  Either one I think is fine, just have them be the same?